bluesky_kafka/mongo_normalized_consumer.py

```diff
@@ -0,0 +1,28 @@
+import msgpack
```

Comment:
PEP8 says imports should be in order of standard library, third party libraries, local libraries:
```python
from functools import partial
import os

import msgpack
import msgpack_numpy as mpn

from bluesky_kafka import MongoBlueskyConsumer
```